app/service/planning_svc.py

```diff
@@ -139,3 +139,11 @@ def __init__(self):
                 if operation.has_fact(trait=adjustment.trait, value=adjustment.value):
                     l.visibility.apply(adjustment)
                     l.status = l.states['HIGH_VIZ']
+
+    async def _apply_global_cleanup_links(self, operation, agent, link_status):
+        links = []
```

Comment:
this block doesn't work as expected. we would want to clean it up a bit as well, as it is messy + it duplicates some fo the above work... likely this block can be combined with the calling function (_generate_cleanup_links)